src/test/java/ay2021s1_cs2103_w16_3/finesse/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,7 @@
     public static final Index INDEX_FIRST_TRANSACTION = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_TRANSACTION = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_TRANSACTION = Index.fromOneBased(3);
+
+    public static final Index INDEX_FIRST_FREQUENT_EXPENSE = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_FREQUENT_EXPENSE = Index.fromOneBased(2);
```

Comment:
These `Index` objects are repeated from those above. I would suggest reusing those above rather than create multiple repeats of the exact same object. Perhaps maybe rename the three indexes above to something more general, like `INDEX_FIRST` and so on.